POS/UserInterface/01b_History.md

```diff
@@ -3,6 +3,7 @@
 *Venduo POS > Sales > Tab HISTORY*
 
 ![History](../../Assets/Screenshots/POS/Sales/History/History.png "[History]")
+<!---Error-->
 
```

Comment:
- [Info] The printer must be configured to print the sales receipt. --> Link zu Integration/Configure printer hinzufügen?